pipelines/h3c/idr2/v1/h3c_idr2_1.toml

```diff
@@ -127,22 +126,32 @@ args = ["{basename}", "${ABSCAL_OPTS:model_files_glob}", "${ABSCAL_OPTS:nInt_to_
         "${ABSCAL_OPTS:phs_conv_crit}", "${ABSCAL_OPTS:edge_cut}"]
 
 [XRFI]
+mem = 96000
 prereqs = "ABSCAL"
-mem = 64000
-args = ["{basename}", "${XRFI_OPTS:kt_size}", "${XRFI_OPTS:kf_size}", "${XRFI_OPTS:sig_init}",
-        "${XRFI_OPTS:sig_adj}"]
+args = ["${XRFI_OPTS:kt_size}", "${XRFI_OPTS:kf_size}", "${XRFI_OPTS:sig_init}",
+        "${XRFI_OPTS:sig_adj}", "{obsid_list}"]
+# xrfi will drop data within kt_size of time edge. Total integrations read in
+# must be greater than 2 * kt_size. With 2 integrations per file, and kt_size=8,
+# n_time_neighbors must be >= 4 (with centered==True).
+# Larger n_time_neighbors will reduce redundant i/o and median filter calculation.
+# The numbers below result in about 35% redundancy.
+n_time_neighbors = 15
+stride_length = 23
+time_centered = true
+collect_stragglers = true
 
 
 [XRFI_DAY_THRESHOLD]
 prereqs = "XRFI"
-time_prereqs = "XRFI"
+args = ["${XRFI_DAY_THRESHOLD_OPTS:nsig_f}", "${XRFI_DAY_THRESHOLD_OPTS:nsig_t}",
+        "${XRFI_DAY_THRESHOLD_OPTS:nsig_f_adj}", "${XRFI_DAY_THRESHOLD_OPTS:nsig_t_adj}",
+        "{obsid_list}"]
 n_time_neighbors = "all"
-args = ["{basename}", "${XRFI_DAY_THRESHOLD_OPTS:nsig_f}", "${XRFI_DAY_THRESHOLD_OPTS:nsig_t}",
-        "${XRFI_DAY_THRESHOLD_OPTS:nsig_f_adj}", "${XRFI_DAY_THRESHOLD_OPTS:nsig_t_adj}"]
+stride_length = "all"
+time_centered = false
 
 [CAL_SMOOTH]
 prereqs = "XRFI_DAY_THRESHOLD"
-time_prereqs = "XRFI_DAY_THRESHOLD"
 n_time_neighbors = "all"
 mem = 64000
 args = ["{basename}", "${CAL_SMOOTH_OPTS:freq_scale}", "${CAL_SMOOTH_OPTS:time_scale}",
```

Comment:
I think this will technically still work, but maybe we want to update any script that uses `n_time_neighbors = "all"` to get `obsid_list` passed in?